app/constants/animalsClasses.ts

```diff
@@ -0,0 +1,32 @@
+export const RareAnimalsClasses = [
```

Comment:
Could we add a comment with some pointer to where these classes come from (e.g. link to a repo) or would it be too volatile?

Reply:
done